### PR TITLE
Docker compose for test; Fix keys on delete_topics

### LINKED ISSUE #90: Wrong key for delete_topics
Should this be `topic_names` instead of `topics`? https://github.com/kafka4beam/kafka_protocol/blob/master/src/kpro_req_lib.erl#L360

As per the bnf here https://github.com/kafka4beam/kafka_protocol/blob/master/priv/kafka.bnf#L3781

I'm trying to get tests working in `brod` against the released version of `kafka_protocol` 3.1.0 and I have two tests failing because:

```
{{field_missing,[{stack,[{delete_topics,0},topic_names]},
                 {input,#{timeout => 280000,topics => [<<"brod_SUITE">>]}}]},
```

### PR DESCRIPTION
**Pull request by tpitale**
Resolve #90

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,6 +61,10 @@ hex-publish: distclean
 	$(verbose) rebar3 hex publish
 	$(verbose) rebar3 hex docs
 
+.PHONY: test-env
+test-env:
+	@./scripts/setup-test-env.sh
+
 .PHONY: testbed
 testbed:
 	@$(verbose) ./scripts/setup-testbed.sh $(KAFKA_VERSION)
```
Comment (zmstone) [inline Makefile]: please delete this.

Comment (tpitale) [reply Makefile]: Should I delete all the testbed stuff?

Comment (zmstone) [reply Makefile]: yes please

**Comment by zmstone** [comment]
Thanks for the pr. One last thing. Please help to update changelog.md

**Comment by tpitale** [comment]
@zmstone I wasn't sure which version you would like to use next. Just picked the smallest next version.

**Pull request merged**